neonvm-kernel/README.md

```diff
@@ -26,18 +26,19 @@ with the following sequence of actions:
    ```
 2. Then, inside the container, run:
```

Comment:
1. Should it be explained in this section that you need to do upgrade twice: once for x86, and once for arm64?
2. Would it be possible to do this:
```
export KERNEL_ARCH=...
export OLD_VERSION=...
export NEW_VERSION-...
```
in the first step, and then pass the variables to `docker run` with `--env`, and overall make it so the remaining commands don't need to be modified?